test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -64,47 +64,6 @@ int constexpr roundTo32(int _num)
 	return (_num + 31) / 32 * 32;
 }
 
-BOOST_AUTO_TEST_CASE(transaction_status)
-{
-	char const* sourceCode = R"(
-		contract test {
-			function f() public { }
-			function g() public { revert(); }
-			function h() public { assert(false); }
-		}
-	)";
-	compileAndRun(sourceCode);
-	callContractFunction("f()");
-	BOOST_CHECK(m_transactionSuccessful);
-	callContractFunction("g()");
-	BOOST_CHECK(!m_transactionSuccessful);
-	callContractFunction("h()");
-	BOOST_CHECK(!m_transactionSuccessful);
-}
-
-
-BOOST_AUTO_TEST_CASE(smoke_test)
-{
-	char const* sourceCode = R"(
-		contract test {
-			function f(uint a) public returns(uint d) { return a * 7; }
-		}
-	)";
-	compileAndRun(sourceCode);
-	testContractAgainstCppOnRange("f(uint256)", [](u256 const& a) -> u256 { return a * 7; }, 0, 100);
```

Comment:
By the way: for this particular test here, it's not a problem and I don't really mind, but this command *actually* tests every value between ``0`` and ``100``, whereas the extracted test only checks two values, namely ``0`` and ``99``.

So actually tests involving ``testContractAgainstCppOnRange`` should probably stay in ``SolidityEndToEndTest.cpp``.

But yeah, in this particular case, it really doesn't matter, if the full range is checked or not, so it's fine.

Reply:
Ok. I will keep in my mind for future changes